src/pages/index.tsx

```diff
@@ -16,30 +19,8 @@ export const getStaticProps = async () => {
     console.log(err);
   }
 };
-export interface Props {
-  data: [
-    {
-      name: {
-        common: string;
-        official: string;
-        nativeName?: {};
-      };
-      population: number;
-      region: string;
-      subregion: string;
-      capital: [];
-      currencies: {};
-      languages: {};
-      tld: [];
-      borders: [];
-      flags: {
-        svg: string;
-      };
-      cca3: string;
-    }
-  ];
-}
-const Home: NextPage<Props> = ({ data }) => {
+
+const Home: NextPage<{data: CountryInterface[]}> = ({ data }) => {
```

Comment:
rather than an inline type, you can also create a prop here. For consistency, go with the following format
```
type = ComponentProps {};

const Component = ({ someProp }: ComponentProps): ReactElement => {};

```